src/components/EditListingDescriptionPanel/EditListingDescriptionPanel.js

```diff
@@ -24,48 +28,105 @@ const EditListingDescriptionPanel = props => {
     panelUpdated,
     updateInProgress,
     errors,
+    listingType,
   } = props;
 
   const classes = classNames(rootClassName || css.root, className);
   const currentListing = ensureOwnListing(listing);
   const { description, title, publicData } = currentListing.attributes;
+  const { manufactureYear, maxUsingTime, equipmentType } = publicData;
 
   const isPublished = currentListing.id && currentListing.attributes.state !== LISTING_STATE_DRAFT;
+  const renderMessageByListingType = () => {
+    if (listingType === LISTING_TYPE.equipment)
+      return <FormattedMessage id="EditEquipmentListingDescriptionPanel.createListingTitle" />;
+    else return <FormattedMessage id="EditListingDescriptionPanel.createListingTitle" />;
+  };
+
   const panelTitle = isPublished ? (
     <FormattedMessage
       id="EditListingDescriptionPanel.title"
       values={{ listingTitle: <ListingLink listing={listing} /> }}
     />
   ) : (
-    <FormattedMessage id="EditListingDescriptionPanel.createListingTitle" />
+    renderMessageByListingType()
   );
 
-  const categoryOptions = findOptionsForSelectFilter('category', config.custom.filters);
+  const getCategoryOptions = () => {
+    if (listingType === LISTING_TYPE.equipment) {
+      return findOptionsForSelectFilter('equipmentType', config.custom.filters);
+    } else {
+      return findOptionsForSelectFilter('category', config.custom.filters);
+    }
+  };
+
+  const renderFormByListingType = () => {
```

Comment:
Why it is a function instead of variable?

and also 
we can use

const renderMessageByListingType = () => listingType === LISTING_TYPE.equipment
? CompA
: CompB;

Or

const renderMessageByListingType = listingType === LISTING_TYPE.equipment
? CompA
: CompB

instead of use if else